app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/SectionedRoutes.java

```diff
@@ -0,0 +1,58 @@
+package ithaca_transit.android.cornellappdev.com.ithaca_transit.Models;
+
+import java.util.HashMap;
+
+public class SectionedRoutes {
+    private static final String SECTION_OPTIMAL = "optimal";
+    private static final String SECTION_FROM_STOPS = "fromStops";
+    private static final String SECTION_BOARDING_SOON = "boardingSoon";
+    private static final String SECTION_WALKING = "walking";
+
+    private Route[] fromStop;
+    private Route[] boardingSoon;
+    private Route[] walking;
+
+    /* Represented as a list (of size 1) so that it can be made into a section in the Routes Options
+       Sectioned Recycler View
+    */
```

Comment:
these comments should look like:

```
/** 
 * ...
 */
```

However, I think having a normal `//` comment is correct here